sources/ProfilesSessionPreferencesViewController.m

```diff
@@ -110,6 +111,10 @@ - (void)awakeFromNib {
                            key:KEY_IDLE_CODE
                           type:kPreferenceInfoTypeIntegerTextField];
     info.range = NSMakeRange(0, 256);
+    
+    [self defineControl:_idlePeriod
+                           key:KEY_IDLE_PERIOD
+                          type:kPreferenceInfoTypeDoubleTextField];
```

Comment:
This should be disabled just like `_idleCode`. See how it uses the `observer` block on line 106.